server/models/Event.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose");
+
+const eventSchema = new mongoose.Schema({
+  userURL: {
+    type: String,
+    required: true,
+  },
+  eventDuration: {
+    type: Number,
+    required: true
+  },
+  eventDate: {
+    type: Date,
+    required: true
+  },
+  // eventTimeZone:{
+  //   type: Date,
+  // }
```

Comment:
If this isn't needed we don't need to push this into production

Reply:
My bad, thanks